### DISCUSSION TITLE
Errors while running automated tests

### DISCUSSION THREAD

**Discussion by shivabhardwaj18** [pytest-dev/pytest-bdd#692] [Q&A]
Hello,

I have been getting following error when I am using newer versions of pytest-bdd:

<img width="947" alt="image" src="https://github.com/pytest-dev/pytest-bdd/assets/76955043/99f69c9f-de8f-4e30-9cc8-5a3ba1864ed1">

If someone can help me how to resolve it please? I have tried various new versions of pytest-bdd (including latest one now - 7.2.0).

Thanks in advance,
Shiva Bhardwaj

**Comment by youtux** [comment]
`example_converters` param was removed. Please check the migration guide: https://pytest-bdd.readthedocs.io/en/stable/#migration-of-your-tests-from-versions-4-x-x
